pkg/commands/process/settings/rules/javascript/express/external_file_upload.yml

```diff
@@ -0,0 +1,21 @@
+patterns:
+  - pattern: res.sendFile($<EXPRESS_REQ>)
+    filters:
+      - variable: EXPRESS_REQ
+        detection: javascript_express_request_obj
+languages:
+  - javascript
+auxiliary:
+  - id: javascript_express_request_obj
+    patterns:
+      - req.$<_>
+trigger: presence
+severity:
+  default: "warning"
+metadata:
+  description: "Do not allow user input to control filename or path."
```

Comment:
We should now follow the convention from https://github.com/Bearer/bearer/pull/627